.github/workflows/stale.yml

```diff
@@ -20,17 +20,17 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         ascending: true # Since we have so many issues, the stale bot finds it hard to keep track. This makes sure that at least the oldest are removed.
         # Issue config
-        stale-issue-message: There has been no activity on this issue in three months. It will be closed in a week if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback.
+        stale-issue-message: There has been no activity on this issue in four weeks. It will be closed in a week if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback.
         close-issue-message: This issue was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback.
-        days-before-issue-stale: 90
+        days-before-issue-stale: 28
         days-before-issue-close: 7
         exempt-issue-labels: discussions-to,long-term
         stale-issue-label: stale
         # PR config
-        stale-pr-message: There has been no activity on this pull request in two months. It will be closed in a week if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback or request a review in a comment.
+        stale-pr-message: There has been no activity on this pull request in a week. It will be closed in two weeks if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback or request a review in a comment.
         close-pr-message: This pull request was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.
-        days-before-pr-stale: 60
-        days-before-pr-close: 7
+        days-before-pr-stale: 7
```

Comment:
I think this is too aggressive. I only review PRs once a week, and I've spent my time the last couple weeks making `eipw`.

Reply:
How about two weeks?